README.md

```diff
@@ -1,144 +1,153 @@
-# ember-cli-head [![Build Status](https://github.com/ronco/ember-cli-head/workflows/Build/badge.svg?branch=master)](https://github.com/ronco/ember-cli-head/actions?query=branch%3Amaster+workflow%3A%22Build%22)
+[![Build Status](https://github.com/ronco/ember-cli-head/workflows/Build/badge.svg?branch=master)](https://github.com/ronco/ember-cli-head/actions?query=branch%3Amaster+workflow%3A%22Build%22)
 
-This addon adds easy population of head tags from your Ember code
-without any direct hacky DOM manipulation.  This addon also provides
-[ember-cli-fastboot](https://github.com/tildeio/ember-cli-fastboot)
-compatibility for generating head tags in server-rendered apps.
+# ember-cli-head
+
+This addon lets you populate `<head>` tag from your Ember code without any direct hacky DOM manipulation. It also provides [ember-cli-fastboot](https://github.com/ember-fastboot/ember-cli-fastboot) compatibility for generating head tags in server-rendered apps.
+
+The hope is that, in the future, Ember will provide a mechanism for populating
+`<head>` tag from your app. Until then, this addon provides that functionality.
 
-The hope is that Ember itself will provide a mechanism for populating
-head tags from your app at some time in the future.  Until then this
-addon provides that functionality.
 
 ## Compatibility
 
 * Ember.js v2.18 or above
 * Ember CLI v2.13 or above
 * Node.js v10 or above
 
+
 ## Installation
 
 Install by running
 
-```
+```bash
 ember install ember-cli-head
 ```
 
-And add `{{head-layout}}` to the top of your application template.
+Then, add `<HeadLayout />` to the top of your application template.
+
+```handlebars
+{{!-- app/templates/application.hbs --}}
+
+<HeadLayout />
+
+{{outlet}}
+```
+
+
+### Version
+
+Take into account that version >= 0.3 of this addon require Ember 2.10+ and fastboot >=1.0.rc1. Please use 0.2.X if you don't fulfill both requirements.
```

Comment:
Since the current version of `ember-cli-head` supports Ember 2.18+ and the current version of FastBoot is 2.x, I wondered if this subsection could be removed. Just in case, I kept the subsection.